source/administration/object-management.rst

```diff
@@ -99,7 +99,7 @@ Clients therefore drive the overall hierarchy of data within a given bucket or p
 
 .. cond:: windows
 
-   Unlike filenames on a Windows system, object names in MinIO cannot have a ``\`` character.
+   Unlike filenames on a Windows system, object names in MinIO cannot have either a ``\`` or ``:`` character in the object name.
```

Comment:
maybe a bit more obvious?
```suggestion
   Unlike filenames on a Windows system, object names in MinIO must not include either ``\`` or ``:`` characters in the object name.
```

Reply:
I think we're backing into this. Suggestion:

```suggestion
MinIO does not support ``\`` or ``:`` characters in object names, regardless of support for those characters in Windows filesystems.
```

Reply:
Sure!